### PR TITLE
10706 move all incubator modules of elico8 to public GitHub product cost

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: product_cost/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization
```
Comment (elicoidal) [inline product_cost/README.rst]: Correct headers/README/openerp.py/PEP8 and publish to OCA

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @seb-elico @duanyp1991

**Comment by elicoidal** [comment]
LGTM

**Comment by siyuancn** [comment]
@liulixia1990 Can you have a look? Is the module a last version?

**Comment by liulixia1990** [comment]
@siyuancn  Yes, it's the latest version.

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**